setup.py

```diff
@@ -4,7 +4,7 @@
 
 install_requires = [
     "graphql-core>=3.1,<3.2",
-    "yarl>=1.4,<1.6",
+    "yarl>=1.4,<2.0",
```

Comment:
Actually it should be `yarl>=1.6,<2.0` based on https://github.com/aio-libs/aiohttp/pull/4973